book/src/guide/specifications.md

```diff
@@ -155,7 +228,59 @@ with `const` generics
 
 ### Opaque Structs
 
-See the [specifications guide](specs.md) for more about the `opaque` annotation.
+Flux offers an attribute `opaque` which can be used on structs. A module defining an opaque struct should define a trusted API, and clients of the API should not access struct fields directly. This is particularly useful in cases where users need to define a type indexed by a different type than the structs fields. For example, `RMap` (see below) defines a refined HashMap, indexed by a `Map` - a primitive sort defined by flux.
+
+```rust
```

Comment:
I think by default, `rust` blocks have a "play" button (which sends the query to goto). I don't know if we want some of these blocks to have the button.

Reply:
Right, let me leave that to "future work" -- I mashed together the old `specs.md` and `use.md` here, but yes, I think its a good idea to also make these parts clickable.

Reply:
I meant the opposite. To make some of them not clickable. By default they all get the play button (I think) and you have to annotate them with `noplayground` for it to just be text. Or did your javascript changes completely change the way blocks work?